manifests/init.pp

```diff
@@ -110,18 +109,18 @@
       require   => $nfs_requires,
     }
 
-    if "${::osfamily}-${::operatingsystemrelease}" =~ /^Suse-11/ {
+    if include_sysconfig {
```

Comment:
Found a bug!

@Phil-Friderici  We're missing a dollar sign here ($) to indicate it's a variable?

Interesting that the spec tests didn't catch it though. Found it by testing it on a real system and something didn't add up.

Reply:
Thanks for finding this ! I've fixed the code and the tests accordingly.
Maybe I should rethink my approach to usually not check for not existing resources ;)

Have a great weekend!!!